src/rewrite_app_repos.sh

```diff
@@ -26,7 +24,16 @@ build_assets() {
   branch="$1"
   reponame="$2"
 
-  git clone --recurse-submodules --single-branch --branch "${branch}" https://github.com/greenpeace/"${reponame}"
+  # If the repository name is the repository of the current job, the code will already be checked out, even if it's a
+  # fork PR.
+  if [ "$CIRCLE_PROJECT_REPONAME" == "$reponame" ]; then
```

Comment:
I could add a check here (and [here](https://github.com/greenpeace/planet4-builder/pull/45/files#diff-54216e5f4228fa168f9fa147150a842bR73)) to see if the directory is actually there. That way we can merge this PR before merging theme and plugin PR.